src/models/xgboost.py

```diff
@@ -31,3 +31,11 @@ def forward(self, xs, ys):
 
         return torch.stack(preds, dim=1)
 
+class XGBoostModelSGN(XGBoostModel):
+    def __init__(self, **kwargs):
+        super(XGBoostModelSGN, self).__init__()
+        self.name = "xgboostSGN"
+        self.context_length = -1
+
+    def forward(self, xs, ys):
+        return super().evaluate(torch.sign(xs), ys)
```

Comment:
add a line to the end of this file.

also I get the following warning:

Cannot access member "evaluate" for type "object"

is this something we need to fix or can we just add pyright ignore [...]